sources/constraint-definition.asciidoc

```diff
@@ -436,7 +436,7 @@ If left undefined, the default value for [methodname]`@OverridesAttribute.name`
 
 [NOTE]
 ====
-[tck-testable]#A composing constraint can itself be a composed constraint. In this case, attribute values are overridden recursively according to the described rules.# Note however, that a forwarding rule (as defined by [classname]`@OverridesAttribute`) is only applied to the direct composing constraints.
```

Comment:
Ah, got you. I think it becomes more clear that it was meant as I understood it (and as it was written originally) when looking at the next sentences which describe the attribute overriding. I'll exclude that specific change and merge the PR.